.eslintrc.json

```diff
@@ -13,6 +13,13 @@
     "sourceType": "module",
     "ecmaVersion": 6
   },
+  "overrides": [
+    {
+      "files": ["src/**/*.js", "src/**/*.vue"],
+      "excludedFiles": "*.test.js"
```

Comment:
why are tests excluded from eslint?

Reply:
I'm linting for `TODO` statements, but would like to leave them in tests as a breadcrumb for me or the next developer.

Reply:
Hmm, I'm worried that this might lead to issues down the line, but 👍 for now

Reply:
I'll try to remember to lint test files again when we have decent test coverage